File: selenium-webdriver-python-unittest/pages/passenger_page.py
from pages.abstract_page import AbstractPage

from selenium import webdriver
from selenium.webdriver.support.ui import Select

class PassengerPage(AbstractPage):

  def enter_first_name(self, passenger_first_name):
    self.driver.find_element_by_name("passengerFirstName").send_keys(passenger_first_name)

  def enter_last_name(self, passenger_last_name):
    self.driver.find_element_by_name("passengerLastName").send_keys(passenger_last_name)

  def click_next(self):
    self.driver.find_element_by_name("passengerLastName").submit() 

File: selenium-webdriver-nodejs-mocha/pages/passenger_page.js
const AbstractPage = require('./abstract_page');

var webdriver = require('selenium-webdriver'),
    By = webdriver.By,
    until = webdriver.until;

class PassengerPage extends AbstractPage {

  constructor(driver) {
    super(driver);
  }

  async enterFirstName(first_name) {
    await this.driver.findElement(By.name("passengerFirstName")).sendKeys(first_name);
  }

  async enterLastName(last_name) {
    await this.driver.findElement(By.name("passengerLastName")).sendKeys(last_name);
  }

  async clickNext() {
    await this.driver.findElement(By.xpath("//input[@value='Next']")).click();
  }
};

module.exports = PassengerPage;
